mlos_bench/db/db_schema_sqlite3_duckdb.sql

```diff
@@ -0,0 +1,83 @@
+--
+-- Copyright (c) Microsoft Corporation.
+-- Licensed under the MIT License.
+--
+-- SQLite3/DuckDB version of the DB schema.
+
+DROP TABLE IF EXISTS trial_telemetry;
+DROP TABLE IF EXISTS trial_results;
+DROP TABLE IF EXISTS trial_config;
+DROP TABLE IF EXISTS experiment_merge;
+DROP TABLE IF EXISTS trial_status;
+DROP TABLE IF EXISTS experiment_config;
+
+CREATE TABLE experiment_config (
+    exp_id VARCHAR(255) NOT NULL,
+    descr TEXT,
+    git_repo VARCHAR(255) NOT NULL,
+    git_commit VARCHAR(40) NOT NULL,
+
+    PRIMARY KEY (exp_id)
+);
+
+CREATE TABLE trial_status (
+    exp_id VARCHAR(255) NOT NULL,
+    trial_id INTEGER NOT NULL,
```

Comment:
```suggestion
    trial_id INTEGER NOT NULL, -- NOTE: NULL used as implicit auto-increment
```

Reply:
Let's not autoincrement trial_id - this leads to very confusing trial numbers within each experiment